### PR TITLE
Fix SkiaSharp broken by 6.0.200 to 6.0.201 sdk not being compatible

### PR DESCRIPTION
**Pull request by danwalmsley**
6.0.200 and 6.0.201/202 sdks have breaking changes.

This means that we now have to update skiasharp to newer builds compiled specifically on sdk 201/202.

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019874-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: src/iOS/Avalonia.iOS/Avalonia.iOS.csproj [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\Avalonia.Base\Avalonia.Base.csproj" />
     <ProjectReference Include="..\..\Skia\Avalonia.Skia\Avalonia.Skia.csproj" />
   </ItemGroup>
-  <Import Project="Sdk.targets" Sdk="Xamarin.Legacy.Sdk" />
-
+  
   <!-- Workaround for https://github.com/dotnet/msbuild/issues/4584 -->
   <Target Name="_RemoveNativeReferencesManifest" AfterTargets="BuiltProjectOutputGroup">
```
Comment (maxkatz6) [inline src/iOS/Avalonia.iOS/Avalonia.iOS.csproj:13]: You can remove this block

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019877-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019880-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019890-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Pull request merged**